src/signalo/urls.py

```diff
@@ -21,5 +21,6 @@
 urlpatterns = [
     path("signalo_core/", include("signalo.core.urls")),
     path("admin/", admin.site.urls),
-    path("oapif/", include(oapif_router.urls)),  # Django-rest urls
+    path("oapif/", include(oapif_router.urls)),  # Django-rest urls,
```

Comment:
```suggestion
    path("oapif/", include(oapif_router.urls)),  # Django-rest urls
```

I guess a comma by accident.